packages/backend-app-api/src/wiring/ServiceRegistry.ts

```diff
@@ -191,6 +191,16 @@ export class ServiceRegistry {
     }
   }
 
+  hasBeenAdded(ref: ServiceRef<any>) {
+    if (ref.id === coreServices.pluginMetadata.id) {
+      return true;
+    }
+    if (ref.multiton) {
```

Comment:
This may be unfortunate to have here right? Unexpected by future callers. Could it be better to add this multiton check on the calling side?

Reply:
Hmm, maybe. It currently mirrors logic from a bit further up. I'll try it out